src/resources/js/tribe-common.js

```diff
@@ -17,13 +17,32 @@ tribe_auto_sysinfo.ajax = {
 	 */
 	my.init_copy = function () {
 
-		new Clipboard( '.system-info-copy-btn' );
+		var clipboard = new Clipboard( '.system-info-copy-btn' );
```

Comment:
Just a thought, no need to change if you don't want to, but we could also write this block in this kind of style:

```
var clipboard   = whatever,
    button_icon = something else,
    button_text = etc;
```

Saves a bit of space, but still makes for clean reading. Admittedly a matter of personal preferences though :smiley: